common/types_test.go

```diff
@@ -158,9 +158,10 @@ func TestMixedcaseAccount_Address(t *testing.T) {
 	// Note: 0X{checksum_addr} is not valid according to spec
 
 	var res []struct {
-		A     MixedcaseAddress
+		A Address
```

Comment:
Is it not possible to keep this a `MixedcaseAddress` type? That would be preferred.

Reply:
Is the problem that the json test case fails to unmarshal? If so, we should just fix the unmarshaller, so this can stay as `MixedcaseAddress`

Reply:
I see you resolved this conversation, but I'm still not clear on the resolution. Can we leave this as `MixedcaseAddress` if the unmarshaller is fixed?